include/PetriEngine/Colored/Reduction/ColoredReducer.h

```diff
@@ -107,6 +106,16 @@ namespace PetriEngine::Colored {
             std::vector<uint32_t> _skippedPlaces;
             std::vector<uint32_t> _skippedTransitions;
 
+            std::vector<ReductionRule *> _usedReductions{};
```

Comment:
I don't think you use this

Reply:
Fixed in [d947046](https://github.com/DAT10CPN/verifypn/pull/3/commits/d947046919a2344a634bd2c9dcfa1ffc7b3bd650)